_docs-sources/pipelines/maintain/drift-detection.md

```diff
@@ -0,0 +1,110 @@
+:::
+
+## What Is Pipelines Drift Detection
+
+Infrastructure Drift occurs when the applied terragrunt cloud configuration differs from the commited Infrastructure as Code (IaC).
+
+Pipelines Drift Detection helps to mitigate Drift in your repositories by running `terragrunt plan` on infrastructure units. If the plan detects any units have drifted from their applied configuration Pipelines will open a Drift Detected Pull Request tracking this drift in your repository.
+
+When the Drift Detected Pull Request is merged, Pipelines will run `terragrunt apply` on all units where drift was detected.
+
+## Installing Pipelines Drift Detection
+
+Pipelines Drift Detection is automatically vended in new Enterprise repositories.
+
+To add Drift Detection to an existing repository:
+
+1. Create a new file at `.github/workflows/pipelines-drift-detection.yml`
+2. Paste in the workflow yml from below
+<details>
+  <summary>pipelines-drift-detection.yml</summary>
+
+  ```yaml
+  name: Pipelines Drift Detection
+  run-name: "[GWP]: Pipelines Drift Detection"
+  on:
+    # Uncomment to enable scheduled Drift Detection
+    # schedule:
+    #  - cron: '0 0 * * 1-5'
+    workflow_dispatch:
+
+  permissions:
+    actions: read
+    id-token: write
+    contents: write
+    # Uncomment the following line in infrastructure-live-root and infrastructure-live-access-control
+    # pull-requests: read
+    # Uncomment the following line in delegated repositories
+    # pull-requests: write
+
+  jobs:
+    GruntworkPipelines:
+      uses: gruntwork-io/pipelines-workflows/.github/workflows/pipelines-drift-detection.yml@v2
+      secrets:
+        PIPELINES_READ_TOKEN: ${{ secrets.PIPELINES_READ_TOKEN }}
+        # Uncomment the following line for your repository:
+        # infrastructure-live-root:
+        # PR_CREATE_TOKEN: ${{ secrets.INFRA_ROOT_WRITE_TOKEN }}
+        # infrastructure-live-access-control:
+        # PR_CREATE_TOKEN: ${{ secrets.ACCESS_CONTROL_WRITE_TOKEN }}
+        # delegated repositories:
+        # PR_CREATE_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  ```
+
+</details>
+3. Uncomment the appropriate line in the permissions block for pull-request permissions
+4. Uncomment the appropriate line in the secrets block for PR_CREATE_TOKEN
+5. For delegated repositories - ensure that **Allow GitHub Actions to create and approve pull requests** is checked in the repository settings.
+
+![Allow Actions To Create Pull Requests](/img/pipelines/maintain/drift_detection_allow_pr_create.png)
+
+
+## Detecting Drift
+
+Pipelines Drift Detection can be run on a schedule or manually.
+
+### Running on a schedule
+
+To enable running on a schedule, uncomment the schedule block containing `- cron: '0 0 * * 1-5'` in `.github/workflows/pipelines-drift-detection.yml`. The default schedule runs at 00:00UTC Monday through Friday. You can increase or decrease the frequency that the schedule runs using [crontab syntax](https://crontab.guru/#0_0_*_*_1-5).
```

Comment:
IMO anytime we give somebody instructions to do things, it should be an ordered sequence of steps, e.g.  

<b>Enabling Drift Detection on a schedule</b>
1. Do X
2. Do Y